### ISSUE TITLE
Can not login with root

### ISSUE THREAD

**Issue by DST73** [ayufan/pve-backup-server-dockerfiles#50]
Could you please post what the initial password for the root use is?

Thank you!

**Comment by redtex** [comment]
https://github.com/ayufan/pve-backup-server-dockerfiles#run

login - admin@pbs
passw - pbspbs

**Comment by DST73** [comment]
Thanks! but I asked for the root users password. Maybe I didn't ask clearly :)

![image](https://github.com/user-attachments/assets/24c2ae01-a6bf-4b6a-9ad8-9427b8d672b5)

**Comment by redtex** [comment]
exec in container ```passwd``` and set root password. There is no default password for root, because it's no real use for it, as mentioned [here](https://github.com/ayufan/pve-backup-server-dockerfiles#features).

**Comment by DST73** [comment]
if I "enter" the container and simply try ls nothing happens. Same with passwd
![grafik](https://github.com/user-attachments/assets/3ff900de-dcb9-4e23-8cb6-601510bc05d9)

**Comment by sojab0on** [comment]
same for me 
doenst matter witch command ls apt passwd keeps blank line

**Comment by ayufan** [comment]
@DST73 

```
docker exec -it ....
```

@sojab0on You should not use `root` as it makes not sense for container-based service.

**Comment by sojab0on** [comment]
> @DST73
> 
> ```
> docker exec -it ....
> ```
> 
> @sojab0on You should not use `root` as it makes not sense for container-based service.

also with admin / pbspbs on booth options it wont login

**Comment by sojab0on** [comment]
> > also with admin / pbspbs on booth options it wont login
> 
> This can only be used to login via web. There's no admin account on shell. You should never use `passwd` on shell as it is lost when re-creating the container.

i meant on the web interface it wont let me login with admin 
constnat login failure 
with logs showing 

authentication failure; rhost=[::ffff:172.16.0.103]:58890 user=admin@pbs@pam msg=open tmpfile in "/run/proxmox-backup/shmem" failed - EISDIR: Is a directory | stdout

**Comment by ayufan** [comment]
Do you have `tmpfs` configured for `/run`?

https://github.com/ayufan/pve-backup-server-dockerfiles/blob/master/docker-compose.yml#L14

**Comment by sojab0on** [comment]
> Do you have `tmpfs` configured for `/run`?
> 
> https://github.com/ayufan/pve-backup-server-dockerfiles/blob/master/docker-compose.yml#L14

yes i used your yml file it , when i do the exec command it goes to the right container. but still admin wont log me in to web interface

**Comment by ayufan** [comment]
@sojab0on Show me `mount`.

**Comment by sojab0on** [comment]
docker_pbs_etc | /etc/proxmox-backup
docker_pbs_lib | /var/lib/proxmox-backup
docker_pbs_logs | /var/log/proxmox-backup

root@944cea7b2e58:/# df -h
Filesystem              Size  Used Avail Use% Mounted on
none                     11T  7.7T  3.3T  71% /
tmpfs                    64M     0   64M   0% /dev
tmpfs                   3.9G     0  3.9G   0% /sys/fs/cgroup
shm                      64M     0   64M   0% /dev/shm
tmpfs                   3.9G  108K  3.9G   1% /run
/dev/mapper/cachedev_0   11T  7.7T  3.3T  71% /etc/hosts
tmpfs                   3.9G     0  3.9G   0% /proc/acpi
tmpfs                   3.9G     0  3.9G   0% /proc/scsi
tmpfs                   3.9G     0  3.9G   0% /sys/firmware

**Comment by ayufan** [comment]
@sojab0on `uname -a`?

**Comment by sojab0on** [comment]
> uname -a

Linux 944cea7b2e58 3.10.108 #42962 SMP Mon Aug 19 15:12:09 CST 2024 x86_64 GNU/Linux

**Comment by sojab0on** [comment]
> OK, this is why. The PBS requires newer kernel: #15.

okay, that why but i have some linux experience but not with compiling or forking to patch it out. 
then pbs on my synology is a no go.

**Comment by ayufan** [comment]
Synology with old kernel is no go for PBS. It requires heavy patching of PBS to make it work on older kernel.
